src/App.js

```diff
@@ -1,26 +1,90 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { useState } from 'react'
 import './App.css';
+import contacts from './contacts.json'
+
+const App = () => {
+  //Iteration 1 | Display 5 Contacts
+  const fiveContacts = contacts.slice(0, 5);
+  const [contactsState, setcontactsState] = useState(fiveContacts);
+
+  //Iteration 2 | Add New Random Contacts
+  const addRandomContacts = () => {
+    const randomContacts = Math.floor(Math.random() * contacts.length);
+    const contactsStateCopy = [...contactsState];
+    contactsStateCopy.push(contacts[randomContacts]);
+    setcontactsState(contactsStateCopy);
+  }
+
+  //Iteration 3 a | Sort Contacts By Name
+  const sortByName = () => {
+    const contactsStateCopy = [...contactsState];
+
+    contactsStateCopy.sort(function (a, b) {
+      if (a.name < b.name) { return -1; }
+      if (a.name > b.name) { return 1; }
+      return 0;
+    });
+    setcontactsState(contactsStateCopy);
+  }
+
+  //Iteration 3 b | Sort Contacts By Popularity
+  const sortByPopularity = () => {
```

Comment:
The popularity is sorted ascending, is this what you would like to happen?

Reply:
Yes, I wanted to see popularity from the lowest to the highest number.